pkg/resolver/related.go

```diff
@@ -261,18 +262,23 @@ func (s *SearchResult) getRelationResolvers(ctx context.Context) []SearchRelated
 	// get uids for related items that match the relatedKind filter.
 	s.filterRelatedUIDs(relatedMap)
 
-	// Build query to get full item data from s.uids
-	s.buildQueryToGetItemsFromUIDs()
-	items, err := s.resolveItems() // Fetch the related items
-	if err != nil {
-		klog.Warning("Error resolving related items.", err)
-		return []SearchRelatedResult{}
-	}
+	// if no relatedKind uids are present - return empty related Search
+	if len(s.uids) > 0 {
```

Comment:
Added this to avoid the error mentioned below:
```
E1102 10:18:14.278177   25927 search.go:253] Error resolving query [SELECT "uid", "cluster", "data" FROM "search"."resources" WHERE ("uid" IN ()) LIMIT 1000] with args [[]]. Error: [ERROR: syntax error at or near ")" (SQLSTATE 42601)]
W1102 10:18:14.278258   25927 related.go:268] Error resolving related items.ERROR: syntax error at or near ")" (SQLSTATE 42601)
```

Reply:
FYI @jlpadilla